standardization/src/test/resources/application.conf

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018-2019 ABSA Group Limited
```

Comment:
Question: Is this one needed (for tests to pass)? I managed to execute them all successfully without this config.
(same with `standardization/src/test/resources/spline.properties`)

Reply:
Empty is OK, IMO.